README.md

```diff
@@ -135,6 +135,19 @@ python generate_custom_image.py \
     customization script. See more information about VM metadata on
     https://cloud.google.com/sdk/gcloud/reference/compute/instances/create.
 
+#### Override properties
+
+Custom images can also be used to overwrite properties set during cluster creation.
```

Comment:
change "overwrite properties" to "overwrite [cluster properties](https://cloud.google.com/dataproc/docs/concepts/configuring-clusters/cluster-properties)"

Reply:
Done.